src/Service.Tests/SqlTests/RestApiTests/Patch/MsSqlPatchApiTests.cs

```diff
@@ -190,6 +215,100 @@ public async Task PatchOneViewBadRequestTest()
             await base.PatchOneViewBadRequestTest(expectedErrorMessage);
         }
 
+        /// <summary>
+        /// Test to validate successful execution of a request when a timestamp field is missing from the request body.
+        /// </summary>
+        [TestMethod]
+        public async Task PatchOneWithTimestampFieldMissingFromRequestBody()
+        {
+            // Validate successful execution of a PATCH update when a timestamp field (here 'row_version')
+            // is missing from the request body.
+            string requestBody = @"
+            {
+                ""book_name"": ""Another Awesome Book"",
+                ""copies_sold"": 100,
+                ""last_sold_on"": null
+            }";
+            string expectedLocationHeader = $"id/1";
+
+            await SetupAndRunRestApiTest(
+                    primaryKeyRoute: expectedLocationHeader,
+                    queryString: null,
+                    entityNameOrPath: _entityWithReadOnlyFields,
+                    sqlQuery: GetQuery("PatchOneUpdateWithTimestampFieldMissingFromRequestBody"),
+                    operationType: EntityActionOperation.UpsertIncremental,
+                    requestBody: requestBody,
+                    expectedStatusCode: HttpStatusCode.OK
+                );
+
+            // Validate successful execution of a PATCH insert when a timestamp field (here 'row_version')
```

Comment:
Here it could be `last_sold_on_date` too right?

Reply:
Yes, we have a test for last_sold_on_date in PatchApiTestBase as it is a computed column and computed columns are read-only both for Pg and MsSql. Thus, that's a common test.